api/v1/storagecluster_types.go

```diff
@@ -537,16 +521,38 @@ const (
 	ConditionVersionMismatch conditionsv1.ConditionType = "VersionMismatch"
 )
 
-// List of constants to show different different reconciliation messages and statuses.
+// SCText stores a StorageCluster reconciliation message or status text
+type SCText int64
+
+// List of constants to show different reconciliation messages and statuses.
 const (
-	ReconcileFailed                 = "ReconcileFailed"
-	ReconcileInit                   = "Init"
-	ReconcileCompleted              = "ReconcileCompleted"
-	ReconcileCompletedMessage       = "Reconcile completed successfully"
-	ExternalClusterConnected        = "ExternalClusterConnected"
-	ExternalClusterConnectedMessage = "Connected successfully to an external cluster"
+	ReconcileFailed SCText = iota
+	ReconcileInit
+	ReconcileCompleted
+	ReconcileCompletedMessage
+	ExternalClusterConnected
+	ExternalClusterConnectedMessage
 )
 
+func (s SCText) String() string {
+	switch s {
+	case ReconcileFailed:
+		return "ReconcileFailed"
+	case ReconcileInit:
+		return "Init"
+	case ReconcileCompleted:
+		return "ReconcileCompleted"
+	case ReconcileCompletedMessage:
+		return "Connected successfully to an external cluster"
+	case ExternalClusterConnected:
+		return "ExternalClusterConnected"
+	case ExternalClusterConnectedMessage:
+		return "Connected successfully to an external cluster"
+
+	}
+	return "unknown"
+}
```

Comment:
Regardless if I agree or disagree with the change, this has nothing to do with the storage profile CR. And if needed should be introduced in a separate unrelated PR